src/integration_tests/integration_utils.py

```diff
@@ -1,3 +1,7 @@
+"""Utilities for integration tests."""
```

Comment:
Not great to have file named `utils.py`, since we have so many `...utils.py` files.